recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+    topics = ("interpolation", "splines", "cubic", "bicubic", "boost", "eigen3")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "5.1",
+        }
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, "14")
+
+        minimum_version = self._minimum_compilers_version.get(
+            str(self.settings.compiler), False
+        )
+        if not minimum_version:
+            self.output.warn(
+                "CubicInterpolation requires C++14. Your compiler is unknown. Assuming it supports C++14."
+            )
+        elif tools.Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                "CubicInterpolation requires C++14, which your compiler does not support."
+            )
+
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("This library is not compatible with Windows")
+
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "cubic_interpolation-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def requirements(self):
+        self.requires("boost/1.75.0")
+        self.requires("eigen/3.3.9")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        _cmake_files = os.path.join(
+            self.package_folder, "lib", "cmake", "CubicInterpolation"
+        )
+        for _f in os.listdir(_cmake_files):
+            os.unlink(os.path.join(_cmake_files, _f))
+        os.rmdir(
+            os.path.join(
+                self.package_folder, "lib", "cmake", "CubicInterpolation"
+            )
+        )
+        os.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.names["cmake_find_package"] = "CubicInterpolation"
        self.cpp_info.names["cmake_find_package_multi"] = "CubicInterpolation"
        self.cpp_info.libs = ["CubicInterpolation"]
```

Reply:
The `cmake_find_package_multi` is marked as experimental in the [docs](https://docs.conan.io/en/latest/reference/generators/cmake_find_package_multi.html). Do I have to check regularly if something has changed and then adjust the required_conan_version, or how is the warning meant? :)